tools/upgrade.sh

```diff
@@ -69,10 +69,16 @@ then
   printf "${BLUE}${BOLD}%s ${UNDER}%s${RESET}\n" "Get your Oh My Zsh swag at:" "https://shop.planetargon.com/collections/oh-my-zsh"
 else
   printf "${RED}%s${RESET}\n" 'There was an error updating. Try again later?'
+
+  # Prepare to return an error exit code
+  status=1
```

Comment:
We can do one better by saving the `$?` immediately after the else, to capture the error code returned by the git command. So:
```suggestion
  status=$?
  printf "${RED}%s${RESET}\n" 'There was an error updating. Try again later?'
```
Remember to update the comment above the exit below.

Reply:
This is much more elegant – I love it!